site/styles/_meta-sub-item.scss

```diff
@@ -47,7 +56,7 @@
     @include media-breakpoint-up(sm) {
       grid-column: auto;
       grid-row: 2;
-      margin-bottom: 10px;
+      margin-bottom: 7px;
```

Comment:
Why not simplify this by removing margins from the title, synopsis, and runtime, and just use the grid gap for consistent 12px spacing between each element?

Reply:
I would like to, but because that is beyond the scope of this task (scope is supposed to be just spacing tweaks/adjustments). I normally would see this as a good opportunity to do things like this, but this card has already blown out too far